CHANGELOG.rst

```diff
@@ -13,6 +13,10 @@ Change Log
 
 Unreleased
 
+[1.42.0] - 2023-07-06
```

Comment:
we also need to update the version here. `1.42.0` -> `1.42.1`

Reply:
Thank you! Copy and paste error, my apologies. Corrected now